src/main/resources/openapi/token-1.0.yaml

```diff
@@ -82,13 +87,15 @@ paths:
           $ref: "#/components/responses/trait_400"
         "500":
           $ref: "#/components/responses/trait_500"
-
   /token/refresh:
     parameters:
       - $ref: headers/okapi-tenant-required.yaml
       - $ref: headers/okapi-url-required.yaml
     post:
-      description: Get a new refresh token and a new access token
+      description: |
+        May be called to obtain a new refresh token and a new access token. Returns a signed, expiring
+        access token and refresh token. Also returns the expiration of each token in the body of the
+        response. The access token time to live is 10 minutes and the refresh token is one week.
```

Comment:
Reword: The time to live is 10 minutes for the access token and one week for the refresh token.